hat/assets/js/apps/Iaso/domains/projects/components/ProjectFeatureFlags.tsx

```diff
@@ -29,14 +31,43 @@ const ProjectFeatureFlags: FunctionComponent<Props> = ({
     featureFlags,
     isFetchingFeatureFlag,
 }) => {
+    const { formatMessage } = useSafeIntl();
     const options = React.useMemo(
         () =>
-            featureFlags?.map(fF => ({
-                label: fF.name,
-                value: fF.id,
-            })),
-        [featureFlags],
+            featureFlags?.map(fF => {
+                return {
+                    label: fF.name,
+                    value: fF.id,
+                    tooltip:
+                        formatMessage(
+                            MESSAGES[`${fF.code.toLowerCase()}_tooltip`],
+                        ) || fF.name,
+                };
+            }),
+        [featureFlags, formatMessage],
+    );
+    const renderOption = (props, option) => (
+        <Tooltip key={option.value} title={option.tooltip} disableInteractive>
+            <span {...props}>{option.label}</span>
+        </Tooltip>
```

Comment:
Please move this outside of the component. The function doesn't depend on any prop or state from the component, so it can safely be moved

Reply:
Done and thank you